### PR TITLE
refactor: Renamed Snake_case class to CamelCase

### PR DESCRIPTION
**Pull request by devaprog**
This PR Fixes Naming of all the classes from Snake case to Camel Case

Issue: #3

### CODE REVIEW

**Review by lakshay-saini-au8 — CHANGES_REQUESTED** [review]
Some URL paths are also changed

**Review by ClawedCatalyst — APPROVED** [review]

**Comment by lakshay-saini-au8** [comment]
@ClawedCatalyst Hey did you checked that some url Paths are also changed

**Comment by devaprog** [comment]
Sorry, I forgot to mention: Work in process

File: authentication/urls.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
     path("login/", Login.as_view()),
-    path("otp_reset_password/", Reset_Password_OTP_View.as_view()),
-    path("enter_new_password/", Password_Change_View.as_view()),
-    path("add_App_password/", Add_Gmail_Pass.as_view()),
-    path("update_app_password/", Update_APP_Password.as_view()),
-    path("get_profile_details/", Profile_details.as_view()),
+    path("otp_reset_password/", ResetPasswordOtpView.as_view()),
+    path("enter_new_password/", PasswordChangeView.as_view()),
+    path("add_App_password/", AddGmailPass.as_view()),
+    path("UpdateAppPassword/", UpdateAppPassword.as_view()),
```
[the remark this answers is not in the document]
Comment (devaprog) [reply authentication/urls.py:15]: Thanks, I got it.

**Comment by devaprog** [comment]
@ClawedCatalyst also could you add `hactoberfest-accepted` label

**Pull request merged**